packages/bento-design-system/src/DateField/Selector.tsx

```diff
@@ -0,0 +1,82 @@
+import { MonthType } from "@datepicker-react/hooks";
+import { useDateFormatter } from "@react-aria/i18n";
+import { FunctionComponent, useMemo } from "react";
+import { Box, Column, Columns } from "../internal";
+import { ListProps } from "../List/createListComponents";
+import { MenuProps } from "../Menu/createMenu";
+import { Label } from "../Typography/Label/Label";
+import { unsafeLocalizedString } from "../util/LocalizedString";
+import { DateFieldConfig } from "./Config";
+import { selector } from "./DateField.css";
+
+function getYears(activeDate: Date): Date[] {
+  const currentYear = new Date().getFullYear();
+  return [...Array(100).keys()].map((diff) => {
+    const yearDate = new Date(activeDate);
+    yearDate.setFullYear(currentYear - diff);
+    return yearDate;
+  });
+}
```

Comment:
sbaglio o facendo così stiamo popolando gli ultimi 100 anni e basta? Includerei forse anche gli anni futuri?

Reply:
giusto 👍

Reply:
non mi è chiarissimo se è più naturale averli in ordine crescente o decrescente. Sarebbe carino anche aprire il menu in focus sull'anno corrente (come fa `react-datepicker` ad esempio), ma il nostro menu/list non lo supporta.